### ISSUE TITLE
Windows 9.1.0 after logout user/password input fields are not accessible for writing

### ISSUE THREAD

**Issue by licaon-kter** [conversejs/converse-desktop#18]
...hitting menu -> reconnect fixes it

**Comment by kicken** [comment]
Switching focus to another application and back also fixes it.  I've not yet been able to determine the cause.

**Comment by kicken** [comment]
I believe I've determined the cause of this, but not yet a good solution.

The cause is the use of the window.confirm method when clicking the logout button.  This move focus away from the web contents in the electron window and the focus is not restored when the dialog is dismissed.

I've tried various combinations of timing and calls to mainWindow.focus(), mainWindow.webContents.focus(), and mainWindow.focusOnWebview() and while some did seem to restore focus, typing in the inputs was still not possible.

The only option that seemed to work reliable was to call mainWindow.blur() then mainWindow.focus(), but that causes the window to flash.

I believe the solution is to either somehow override the use of window.confirm (not sure if possible), or change conversejs to no longer use this function.
